scripts/create_mesh_sample.py

```diff
@@ -31,14 +31,26 @@ def yield_sample_data(data_path, sample_size=SAMPLE_SIZE, random_seed=RANDOM_SEE
             if i in sample_line_indices:
                 yield line
 
-def create_mesh_sample(mesh_data_path, sample_mesh_data_path):
+def create_mesh_sample(mesh_data_path, sample_mesh_data_path, mti_input_path):
+    if mti_input_path:
+        mti_f = open(mti_input_path, "w")
+
     with open(sample_mesh_data_path, "w") as f:
-        for item in yield_sample_data(mesh_data_path):
+        for i, item in enumerate(yield_sample_data(mesh_data_path)):
             f.write(item)
 
+            if mti_input_path:
```

Comment:
Perhaps a comment somewhere with a path to what is expected by MTI and what is output in the e mail? Or maybe in the README, add an example of command line/output for possible reproducibility?  Seems to me that it is as simple as a file with `unique_id|text`.

Reply:
76c76dd